services/src/test/java/io/grpc/services/BinaryLogTest.java

```diff
@@ -203,6 +260,192 @@ public void configBinLog_ignoreDuplicates_method() throws Exception {
     assertEquals(HEADER_256, factory.getLog("p.other2/m"));
   }
 
+  @Test
+  public void callIdToProto() {
+    assertEquals(
+        Uint128
+            .newBuilder()
+            .setHigh(0x1112131415161718L)
+            .setLow(0x19101a1b1c1d1e1fL)
+            .build(),
+        BinaryLog.callIdToProto(CALL_ID));
+
+  }
+
+  @Test(expected = IllegalArgumentException.class)
```

Comment:
`expected` should probably not be used in favor of ExpectedException.  You can't check the error message with expected.

Reply:
done